packages/user-interface/src/pages/TasksPageRequests.mock.tsx

```diff
@@ -0,0 +1,82 @@
+import { FetchResult, GraphQLRequest } from "@apollo/client";
+import { QUERY_GET_TAKEN } from "@nl-portal/nl-portal-api";
+
+const getTaken: {
+  request: GraphQLRequest;
+  result: FetchResult<Record<string, any>>;
+} = {
+  request: {
+    query: QUERY_GET_TAKEN,
+    variables: {},
+  },
+  result: {
+    data: {
+      getTaken: {
+        content: [
+          {
+            id: "cb8e86ab-c58d-11ee-9320-3a659ac0caef",
+            objectId: "d2a2c7d3-af2a-4b0e-8171-24156f84da95",
+            formulier: {
+              formuliertype: "objecturl",
+              value:
+                "https://objecten-zgw.test.denhaag.nl/api/v2/objects/168c8f1f-bf9c-426b-bfcd-b1ad3a202c95",
```

Comment:
ik zou aanraden om hier dummy url's te gebruiken in plaats van urls die overeenkomen met jullie daadwerkelijk url structuur